lib/shopify_api/collection_pagination.rb

```diff
@@ -0,0 +1,52 @@
+module ShopifyAPI
+  module CollectionPagination
+
+    def next_page?
+      next_page_info.present?
+    end
+
+    def previous_page?
+      previous_page_info.present?
+    end
+
+    def fetch_next_page
+      fetch_page(next_page_info)
+    end
+
+    def fetch_previous_page
+      fetch_page(previous_page_info)
+    end
+
+    private
+
+    AVAILABLE_IN_VERSION = ShopifyAPI::ApiVersion::Unstable.new
+
+    def fetch_page(page_info)
+      return [] unless page_info
+
+      resource_class.where(original_params.merge(page_info: next_page_info))
```

Comment:
We could do `where(page_info: next_page_info)`but we're supporting old versions of ActiveResource, and where [was added only in later versions](https://github.com/rails/activeresource/pull/153).